pkgs/development/tools/misc/gdb/wrapper.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenvNoCC, targetPackages
+, makeWrapper
+, gdb-unwrapped
+, safePaths ? [
+   # $debugdir:$datadir/auto-load are whitelisted by default by GDB
+   "$debugdir" "$datadir/auto-load"
```

Comment:
These shouldn't be needed when you don't pass the with-auto-load-safe-path option.

Reply:
Yes thank you I accidentally added them back when I rebased.